torch/utils/cpp_extension.py

```diff
@@ -26,6 +27,8 @@
 
 from setuptools.command.build_ext import build_ext
 
+log = logging.getLogger(__name__)
+
```

Comment:
Does log.info print by default?

Reply:
The default should be stderr. If printing is also needed we can add:

```python
logging.basicConfig(stream=sys.stdout, level=logging.INFO)
```

Reply:
I ran a really quick script, and it looks like log.info doesn't actually print anything out to the console:
```
import torch
import logging
log = logging.getLogger(__name__)
log.info("testing")
```

Reply:
From reading this issue and looking at the code, it looks like we already guard most print statements with a`verbose` flag. This is not bad and I'm not sure we actually need the logging module. We should probably just make sure that we guard all print statements with it

Reply:
Printing to console we have to add `sys.stdout` to stream. The following snippet prints `INFO:__main__:testing`. In PyTorch code only  few testing code have the following set up. 
```python
import sys
import logging
logging.basicConfig(stream=sys.stdout, level=logging.INFO)
log = logging.getLogger(__name__)
log.info("testing")
```

Reply:
> From reading this issue and looking at the code, it looks like we already guard most print statements with a`verbose` flag. This is not bad and I'm not sure we actually need the logging module. We should probably just make sure that we guard all print statements with it

Happy to close this or add `verbose` to the remaining print statements :)

Reply:
> logging.basicConfig(stream=sys.stdout, level=logging.INFO)

This changes the global logging state right? We shouldn't do that because it'll affect other logging usages in pytorch

Reply:
Yes agree. Probably why this setup is only used once in testing and examples. I will close this issue as it doesn't add much to the original file. 

https://github.com/pytorch/pytorch/blob/fe156456194ed64bdf8b086d469b3643515a2baf/test/jit/fixtures_srcs/generate_models.py#L12